src/test/java/io/stargate/sgv2/jsonapi/api/v1/FindAndUpdateIntegrationTest.java

```diff
@@ -1805,4 +1804,146 @@ private void insertDoc(String docJson) {
         .body("status.insertedIds[0]", not(emptyString()))
         .statusCode(200);
   }
+
+  @Nested
+  class UpdateOneWithMin {
+    @Test
+    public void findByColumnAndMin() {
+      insertDoc(
+          """
+                      {
+                         "_id": "update_doc_min",
+                         "min": 1,
+                         "max": 99,
+                         "numbers": {
+                            "values": [ 1 ]
+                          }
+                       }
+                       """);
+      String updateJson =
+          """
+                      {
+                        "updateOne": {
+                          "filter" : {"_id" : "update_doc_min"},
+                          "update" : {"$min" : {
+                              "min": 2,
```

Comment:
Would be nice to have an integration test that uses `$min` and `$max` with types that aren't numbers. For example, `$min: { foo: 'bar' }`. That should work given the `JsonNodeComparatorTest.java` tests, but I think it's worth checking.

Reply:
Yeah. I did add unit tests for wider range given they are more compact, but yeah I could/should add at least one cross-type, and one non-number IT operation.